languages/w3-total-cache.pot

```diff
@@ -7668,6 +7674,11 @@ msgstr ""
 
 #: inc/options/parts/redis.php:30
 msgid ""
+"Verify the server\'s certificate when connecting via TLS."
+msgstr ""
+
+#: inc/options/parts/redis.php:37
+msgid ""
```

Comment:
Full disclosure: I'm not familiar with how localization is set up for this plugin and I couldn't find any developer documentation that mentioned it, so I made the changes to this file by imitating what I saw elsewhere. 😅